server/views/components/image-gallery/index.njk

```diff
@@ -1,19 +1,10 @@
 <div class="image-gallery">
   {% for item in imageGallery.items %}
-    <div class="image-gallery__item">
-      <figure class="figure figure--max-out-height image-gallery__figure">
-        <picture class="figure__picture figure__image-container">
-          <img class="figure__image image-gallery__image" src="{{ item.contentUrl }}" alt="{{ item.caption }}" />
-          <source media="(min-width: {{ item.width }}px)" srcset="{{ item.contentUrl }}" />
-        </picture>
-
-        <figcaption class="figure__caption figure__caption--no-underline image-gallery__figure-caption">
-          <div class="flex">
-            <div>{% icon 'other/picture' %}</div>
-            <div>{{ item.caption }}</div>
-          </div>
-        </figcaption>
-      </figure>
-    </div>
+  {% set smallWidth = item.width/item.height*204 %}
+  {% set largeWidth = item.width/item.height*640 %}
+  {% set queries = ['(max-width: 600px) ', smallWidth, 'px, ', largeWidth, 'px'] %}
+  <div class="image-gallery__item">
```

Comment:
Is there a better way of creating the queries string? This is a bit ugly.

Also, the smallWidth, largeWidth and media query are related to those found in image-gallery.scss, so any changes need to be made in both places, but don't think there is a way around this.

the sizes attribute only takes widths, but the styling determines heights, so need to calculate widths here.